zetaclient/tss/tss_keysign_manager.go

```diff
@@ -0,0 +1,49 @@
+package tss
+
+import (
+	"sync"
+
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+)
+
+// KeySignManager is a manager to keep track of concurrent keysigns performed by go-tss
+type KeySignManager struct {
+	numActiveMsgSigns int64
+	mu                sync.Mutex
+	Logger            zerolog.Logger
+}
+
+// NewKeySignManager - constructor
+func NewKeySignManager(logger zerolog.Logger) *KeySignManager {
+	return &KeySignManager{
+		numActiveMsgSigns: 0,
+		mu:                sync.Mutex{},
+		Logger:            logger.With().Str("module", "KeySignManager").Logger(),
```

Comment:
Does this mean the logs would two module fields? 
wondering if we should limit it to one module and call this is a submodule or something similar ?